test/unit/middlewares/validation.spec.js

```diff
@@ -0,0 +1,18 @@
+const validationMiddleware = require('~/middlewares/validation')
+const { createError } = require('~/utils/errorsHelper')
+const { BODY_IS_NOT_DEFINED } = require('~/consts/errors')
+const signupValidationSchema = require('~/validation/schemas/signup')
+
+describe('Validation middleware', () => {
+  const schema = validationMiddleware(signupValidationSchema)
```

Comment:
This is not a `schema`, it's supposed to be something like `middlewareToTest` or just `middleware`.

Reply:
Fixed